fe/src/main/java/org/apache/doris/load/loadv2/etl/EtlJobConfig.java

```diff
@@ -0,0 +1,487 @@
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+/** jobconfig.json file format
+ * {
+ * 	"tables": {
+ * 		10014: {
+ * 			"indexes": [{
+ * 			    "index_id": 10014,
+ * 				"columns": [{
+ * 				    "column_name": "k1",
+ * 				    "column_type": "SMALLINT",
+ * 				    "is_key": true,
+ * 				    "is_allow_null": true,
+ * 				    "aggregation_type": "NONE"
+ *                }, {
+ * 					"column_name": "k2",
+ * 				    "column_type": "VARCHAR",
+ * 				    "string_length": 20,
+ * 					"is_key": true,
+ * 				    "is_allow_null": true,
+ * 					"aggregation_type": "NONE"
+ *                }, {
+ * 					"column_name": "v",
+ * 				    "column_type": "BIGINT",
+ * 					"is_key": false,
+ * 				    "is_allow_null": false,
+ * 					"aggregation_type": "NONE"
+ *              }],
+ * 				"schema_hash": 1294206574,
+ * 			    "index_type": "DUPLICATE",
+ * 			    "is_base_index": true
+ *            }, {
+ * 			    "index_id": 10017,
+ * 				"columns": [{
+ * 				    "column_name": "k1",
+ * 				    "column_type": "SMALLINT",
+ * 					"is_key": true,
+ * 				    "is_allow_null": true,
+ * 					"aggregation_type": "NONE"
+ *                }, {
+ * 					"column_name": "v",
+ * 				    "column_type": "BIGINT",
+ * 					"is_key": false,
+ * 				    "is_allow_null": false,
+ * 					"aggregation_type": "SUM"
+ *              }],
+ * 				"schema_hash": 1294206575,
+ * 			    "index_type": "AGGREGATE",
+ * 			    "is_base_index": false
+ *          }],
+ * 			"partition_info": {
+ * 				"partition_type": "RANGE",
+ * 				"partition_column_refs": ["k1"],
+ *              "distribution_column_refs": ["k2"],
+ * 				"partitions": [{
+ * 				    "partition_id": 10020,
+ * 					"start_keys": [-100],
+ * 					"end_keys": [10],
+ * 					"is_max_partition": false,
+ * 					"bucket_num": 3
+ *                }, {
+ *                  "partition_id": 10021,
+ *                  "start_keys": [10],
+ *                  "end_keys": [100],
+ *                  "is_max_partition": false,
+ *  				"bucket_num": 3
+ *              }]
+ *          },
+ * 			"file_groups": [{
+ * 		        "partitions": [10020],
+ * 				"file_paths": ["hdfs://hdfs_host:port/user/palo/test/file"],
+ * 				"file_field_names": ["tmp_k1", "k2"],
+ * 				"value_separator": ",",
+ * 			    "line_delimiter": "\n",
+ * 				"column_mappings": {
+ * 					"k1": {
+ * 						"function_name": "strftime",
+ * 						"args": ["%Y-%m-%d %H:%M:%S", "tmp_k1"]
+ *                   }
+ *              },
+ * 				"where": "k2 > 10",
+ * 				"is_negative": false,
+ * 				"hive_table_name": "hive_db.table"
+ *          }]
+ *      }
+ *  },
+ * 	"output_path": "hdfs://hdfs_host:port/user/output/10003/label1/1582599203397",
+ * 	"output_file_pattern": "label1.%d.%d.%d.%d.%d.parquet",
+ * 	"label": "label0",
+ * 	"properties": {
+ * 	    "strict_mode": false,
+ * 	    "timezone": "Asia/Shanghai"
+ * 	}
+ * }
+ */
+public class EtlJobConfig implements Serializable {
```

Comment:
Why this class should implement `Serializable`?
I think it is OK to convert to and from JSON.

Reply:
etl job failed if not implements Serializable interface. failed message: `org.apache.spark.SparkException: Task not serializable, object not serializable`